int16 arm1
int16 arm2
int16 arm3
int16 arm4
